docs/source/Advanced-Usage.md

```diff
@@ -59,7 +59,7 @@ Such compose files are available under `integrations/`.
 
 
 ## Customize analyzer execution at time of request
-Some analyzers provide the chance to customize the performed analysis based on options that are different for each analyzer.
+Some analyzers provide the chance to customize the performed analysis based on options that are different for each analyzer. Incase, some analyzers require other parameters, you can add them by clicking the button `CUSTOM ANALYZERS CONFIGURATION`.
```

Comment:
Some analyzers provide the chance to customize the performed analysis based on options that are different for each analyzer. This is configurable via the `CUSTOM ANALYZERS CONFIGURATION` button on the scan form or you can pass these values as a dictionary when using the pyintelowl client.

Replace the para's text with this^